core/src/main/java/org/opensearch/sql/utils/DateTimeFormatters.java

```diff
@@ -133,6 +133,30 @@ public class DateTimeFormatters {
           .toFormatter(Locale.ROOT)
           .withResolverStyle(ResolverStyle.STRICT);
 
+  // MMDD
+  public static final DateTimeFormatter DATE_FORMATTER_SINGLE_DIGIT_MONTH =
+      new DateTimeFormatterBuilder()
+          .parseDefaulting(YEAR, 2000)
```

Comment:
I think this pattern shouldn't have year. It is ambiguous between `YYYYMDD` and `YYYYMMD`.
```suggestion
```

Reply:
This is needed as far as I understand. I need it to create a `LocalDate` from what is parsed because there is no `ChronoField.YEAR` that can be parsed from the input for this formatter. For example, if the input string is something like `310` for March 10, 2000 then this formatter is used. If I do not have a default value of 2000 then a parsing exception will be thrown (which will cause an `ExprNullValue` to be returned).

Reply:
Also as far as I have seen, an input with 3 digits (e.g., `310`) is parsed by MySQL as `MDD` not `MMD`, so I don't think this implementation is an issue, but please let me know if it still needs to be fixed.